QRCoder/Builders/Payloads/Implementations/EmailPayload.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace QRCoder.Builders.Payloads.Implementations
+{
+    public class EmailPayload : PayloadBase
```

Comment:
Sample where the ctor only has the email address, and the subject/body are provided by extension methods.  Realistically, I'm thinking that the required or common parameters always appear in the constructor (or as an overload), but all of the optional/rare options (like `MailEncoding`) are extension methods.